content/docs/SUSHI/API/_index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "API"
+weight: 25
+---
+
+SUSHI exposes a `fshToFhir` function that can be used to convert FSH strings to FHIR JSON.
+
+## Syntax
+```javascript
+fshToFhir(fshString[, options])
+```
+
+## Parameters
+`fshString` - A string containing FHIR Shorthand definitions.
+
+`options` - An object which can have any combination of the following attributes:
+* `canonical` - A string used to specify the canonical URL.
+* `version` - A string used to specify the version.
+* `fhirVersion` - A string used to specify the version of FHIR to use.
```

Comment:
Do you think there is any benefit to specifying that this should be R4, maybe R5? Or do you think the documentation we have in the SUSHI configuration covers that?